package.json

```diff
@@ -52,15 +51,15 @@
   },
   "devDependencies": {
     "code": "^1.4.1",
-    "eslint": "^2.10.2",
-    "eslint-config-semistandard": "^5.0.0",
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-standard": "^1.3.1",
-    "lab": "^5.11.1",
+    "eslint-config-semistandard": "6.0.2",
+    "eslint-config-standard": "5.3.5",
+    "eslint-plugin-promise": "1.3.2",
+    "eslint-plugin-standard": "1.3.3",
+    "lab": "10.9.0",
```

Comment:
originally upgraded this to see if my problem with building a web driver in `beforeEach` would be fixed (it wasn't) but when other lab changes weren't needed, I kept it. upgrades eslint configs should give us more ES2015 goodness